EIPS/eip-FOCIL.md

```diff
@@ -61,6 +61,18 @@ This EIP introduces backward incompatible changes to the block validation rule s
 
 ## Security Considerations
 
+### Consensus Liveness
+
+The builder or proposer of slot `n+1` cannot construct a canonical block without the inclusion lists and inclusion list aggregate of slot `n`. This protocol relies on proposers being sufficiently peered with the inclusion list committee. Builders (if using MEV-boost) are sufficiently peered with the inclusion list aggregation network.
+
+### Block Construction Time
+
+The builder or proposer of slot `n+1` depends on the inclusion list aggregate of slot `n`. If the inclusion list aggregate appears at the `9s` mark, the builder will have `3s` to update the final block to ensure it meets the requirements. It's crucial for the protocol that `3s` is enough for a solo proposer.
```

Comment:
It is important to ensure there is enough time between the local IL freeze deadline (`t=9` during slot `n`) and the moment at which the block producer has to broadcast `block B` including the IL aggregate, so that:
- There is enough time to update `block B`'s execution payload according to the IL aggregate constraints
- The proposer has enough time to collect and include all local ILs before broadcasting the IL aggregate